parlai/core/worlds.py

```diff
@@ -623,8 +611,11 @@ def display(self):
         s += ("[--end of batch--]")
         return s
 
-    def __len__(self):
-        return len(self.world)
+    def num_examples(self):
+        return math.ceil(sum(w.num_examples() for w in self.worlds) / len(self.worlds))
```

Comment:
hmm, what about it if it isnt defined for one of them?

Reply:
note that this should be `self.world.num_examples()` but jason's question still applies.

currently we'll just get some kind of error about num_examples not being defined, but we could add a new `try/except` saying specifically that it's either not implemented or not available for some of the tasks.

Reply:
one solution is that if one of them returns something that means 'unknown' (via None? 0?) hen the whole thing becomes equal to that value?